content/e/ptw/2024-02-03-ptw-5-gold-rush.md

```diff
@@ -64,6 +64,7 @@ Polish celebrity and former boxer Marcin Najman was also expected to appear in s
   - '[Bad Bones](@/w/bad-bones.md), Marcin "Rzeźnik" Rzeźniczek'
   - g: true
     s: "Pawłowski's in-ring proposal"
+    x: [2] # Exclude Karolina from talent list
```

Comment:
List of 1-based indices, in the order they are listed. 1 is Pawłowski, 2 Karolina, 3 Longman, 4 Bones, 5 Rzeźnik.